src/DynamoCoreWpf/Views/PackageManager/PackageManagerSearchView.xaml.cs

```diff
@@ -29,6 +29,7 @@ protected override void OnClosing(System.ComponentModel.CancelEventArgs e)
         {
             (this.DataContext as PackageManagerSearchViewModel).RequestShowFileDialog -= OnRequestShowFileDialog;
             base.OnClosing(e);
+            this.Owner.Focus();
```

Comment:
I believe adding this `Focus()` method will fix the window issue; thanks for that! Just one small thing: to make the behavior consistent, let's put the call to the `base` method at the end of this overriding method. (1) Unregister the registered events, (2) pass the focus to the owner window, and then (3) close this window.